addon/components/polaris-drop-zone.js

```diff
@@ -477,31 +539,40 @@ export default Component.extend(ContextBoundEventListenersMixin, {
       return;
     }
 
+    let dropzoneContainer = this.element.querySelector('.Polaris-DropZone');
```

Comment:
Since we're using this in a couple of places, how do you feel about making a `dropzoneContainer` CP?

Reply:
I think we're setting this as `this.node` (in `setupNode()`) before this runs, so we should just use that everywhere else?